src/test/java/org/springframework/data/couchbase/domain/Address.java

```diff
@@ -15,7 +15,6 @@
  */
```

Comment:
Add one of our @JsonValue Enums to Address which gets used in the FLE tests and elsewhere.